src/components/templates/PartyMap/components/Map/Map.tsx

```diff
@@ -249,8 +249,8 @@ export const Map: React.FC<PropsType> = ({
               gridTemplateRows: `repeat(${rows}, 1fr)`,
             }}
           >
-            {showGrid && mapGrid}
-            {showGrid && partygoersOverlay}
+            {mapGrid}
+            {partygoersOverlay}
```

Comment:
This is fine because the hooks are already using showGrid and handling this for us. This was a redundant check.